resources/features/Sic.feature

```diff
@@ -0,0 +1,243 @@
+    Given input:
+      | ern | lurn      | sic07 | employees |
+      | 2   | 220000004 | 26400 | 30        |
+      | 2   | 220000005 | 26120 | 20        |
+      | 2   | 220000003 | 26120 | 20        |
+    When the Sic method is calculated
+    Then the Sic results table is produced:
+      | ern | sic07 |
+      | 2   | 26120 |
+
+  @JVM
+    Examples:
+      | language |
+      | Scala    |
+
+  @Happy Path
+
+  Scenario Outline: Happy Path - C - Simple Split (Employees Tied)
+    Given input:
+      | ern | lurn      | sic07 | employees |
+      | 3   | 220000006 | 26400 | 25        |
+      | 3   | 220000007 | 18201 | 25        |
+    When the Sic method is calculated
+    Then the Sic results table is produced:
+      | ern | sic07 |
+      | 3   | 18201 |
+
+  @JVM
+    Examples:
+      | language |
+      | Scala    |
+
+  @Happy Path
+
+  Scenario Outline: Happy Path - D - Complex Split - Div 46
+    Given input:
+      | ern | lurn      | sic07 | employees |
+      | 4   | 220000008 | 46210 | 20        |
+      | 4   | 220000009 | 46310 | 15        |
+      | 4   | 220000010 | 46610 | 25        |
+      | 4   | 220000011 | 46900 | 50        |
+      | 4   | 220000012 | 46110 | 40        |
+    When the Sic method is calculated
+    Then the Sic results table is produced:
+      | ern | sic07 |
+      | 4   | 46610 |
+
+  @JVM
+    Examples:
+      | language |
+      | Scala    |
+
+  @Happy Path
+
+  Scenario Outline: Happy Path - E - Complex Split - Div 47
+    Given input:
+      | ern | lurn      | sic07 | employees |
+      | 5   | 220000013 | 47190 | 35        |
+      | 5   | 220000014 | 47210 | 10        |
+      | 5   | 220000015 | 47300 | 20        |
+      | 5   | 220000016 | 47110 | 35        |
+      | 5   | 220000017 | 47810 | 30        |
+      | 5   | 220000018 | 47910 | 10        |
+      | 5   | 220000019 | 47990 | 40        |
+      | 5   | 220000020 | 47220 | 30        |
+      | 5   | 220000021 | 47220 | 20        |
+    When the Sic method is calculated
+    Then the Sic results table is produced:
+      | ern | sic07 |
+      | 5   | 47220 |
+
+  @JVM
+    Examples:
+      | language |
+      | Scala    |
+
+  @Happy Path
+
+  Scenario Outline: Happy Path - F - Simple Split (SIC Aggregation with Employees Tied)
+    Given input:
+      | ern | lurn      | sic07 | employees |
+      | 6   | 220000022 | 26120 | 20        |
+      | 6   | 220000023 | 26400 | 30        |
+      | 6   | 220000024 | 18201 | 10        |
+      | 6   | 220000025 | 18201 | 10        |
+      | 6   | 220000026 | 26120 | 20        |
+      | 6   | 220000027 | 46210 | 40        |
+    When the Sic method is calculated
+    Then the Sic results table is produced:
+      | ern | sic07 |
+      | 6   | 26120 |
+
+  @JVM
+    Examples:
+      | language |
+      | Scala    |
+
+  @Sad Path
+
+  Scenario Outline: Sad Path - We have an invalid input field
+    Given input:
+      | id  | sic07 | Invalid |
```

Comment:
The input data should match the format of previous examples used, less for the invalid field - 'id' is itself invalid as the method would be expected to use 'ern' and 'lurn' as the keys.